src/components/navBar/demoBasicUsage/script.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+  'use strict';
+
+  angular.module('navBarDemoBasicUsage', ['ngMaterial'])
+      .controller('AppCtrl', AppCtrl);
+
+  function AppCtrl($scope) {
+    $scope.currentLink = 'page1';
+    $scope.goto = function(hash) { $scope.currentLink = hash; };
```

Comment:
Just wanted to confirm: this is only to side-step the actual routing problem mentioned in the index.html comment?